src/components/Calculator.js

```diff
@@ -1,47 +1,49 @@
-import React from 'react';
+import { useState } from 'react';
 import './style.css';
-import AppTitle from './AppTitle';
+import CalculatorTitle from './AppTitle';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [display, setdisplay] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
```

Comment:
- [ ] Well done on implementing `useState`. 👍 But you have missed using props. As one of the main purposes of this project is to learn and use `props` and `useState` so, kindly use `props` with `useState` to grasp the understanding of the previous lessons.
-  Bellow I am giving a suggestion on how you can make the best use of `props` with `useState`.
  - First, you would need to create a JavaScript object containing the calculator's initial state. And you can do it in `App.js`, Pass the data to `/components/Calculator.js`.
  ```
function App() {
    const initialState = {
        total: null,
        next: null,
        operation: null,
      };
  return (
    <div>
        <Calculator initialState={initialState} />
    </div>
  );
}
  ```
  - Then, you can use props `initialState` in the useState like this:
  ```
function Calculator({ initialState }) {
  const [display, setdisplay] = useState(initialState);
  ```

#### NOTE: Here I have just given you an example. It is not must to follow the same way. But use props in the best way you can.